src/lib/Engineer.js

```diff
@@ -0,0 +1,46 @@
+//import employee
+const Employee = require("./Employee");
+
+class Engineer extends Employee {
+  constructor({ github, name, id, email }) {
```

Comment:
Would be nice to implement the `...` rest operator here. Since we only really need to separate the `github` key we can do:

```
constructor( { github, ...rest } ) { 
      super( rest );
      this.github = github

```

Please note - `rest` can be named however you want, this is personal choice.